src/wrapped/wrappedgobject2.c

```diff
@@ -833,6 +845,28 @@ EXPORT void my_g_closure_remove_finalize_notifier(x64emu_t* emu, void* closure,
     my->g_closure_remove_finalize_notifier(closure, data, findGClosureNotify_Fct(f));
 }
 
+EXPORT my_GClosure_t* my_g_closure_ref(x64emu_t* emu, my_GClosure_t* closure)
+{
+    my_GClosure_t* ret = my->g_closure_ref(closure);
+    bridge_t* system = emu->context->system;
+
+    #define GO(A, W)\
+    if(closure->A)      \
+        if(!CheckBridged(system, closure->A)) \
+            AddAutomaticBridge(system, W, closure->A, 0, #A); \
+
+    GO(marshal, vFppuppp)
+    #undef GO
+
+    return ret;
+}
+
+EXPORT void my_g_closure_sink(x64emu_t* emu, my_GClosure_t* closure)
```

Comment:
I don't think it's needed. A simple `GO` type of wrapping for sink should be enough. It doesn't hurt, I just think it's not usefull.